### ISSUE TITLE
Add one-to-one relationship tutorial to docs

### ISSUE THREAD

**Issue by fkromer** [tiangolo/sqlmodel#381]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the SQLModel documentation, with the integrated search.
- [X] I already searched in Google "How to X in SQLModel" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to SQLModel but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to SQLModel but to [SQLAlchemy](https://github.com/sqlalchemy/sqlalchemy).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
n.a.
```

### Description

It would be great to have an "how to" about one-to-one relationships directly in the official docs. An adaption of the sqlalchemy docs example about one-to-on relationships (https://docs.sqlalchemy.org/en/14/orm/basic_relationships.html#one-to-one) should do the job.

### Operating System

Linux

### SQLModel Version

n.a.

### Python Version

n.a.

**Comment by Beafowl-Pull** [comment]
you just need to make a simple relation like here: https://sqlmodel.tiangolo.com/tutorial/relationship-attributes/

**Comment by m0wer** [comment]
Not really. Since there's no option `uselist=False` for the relationship, you always get a list (so it's one-to-many) even if not possible (if you make the FK the PK).

**Comment by Kyu** [comment]
"workaround" for future readers: https://github.com/tiangolo/sqlmodel/issues/132
